src/Tizen.NUI.Components/Controls/Pagination.cs

```diff
@@ -104,9 +95,7 @@ public Size IndicatorSize
         /// <summary>
         /// Gets or sets the background resource of indicator.
         /// </summary>
-        /// <since_tizen> 6 </since_tizen>
-        /// This will be public opened in tizen_5.5 after ACR done. Before ACR, need to be hidden as inhouse API.
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 8 </since_tizen>
         public Selector<string> IndicatorImageURL
```

Comment:
Please use the PascalCase naming convention for the acronyms. Also why to Uri return type?
```suggestion
        public Selector<string> IndicatorImageUrl
```

Reply:
Changed the name as you suggested.
But, in this sentence "why to Uri return type?", what do you mean?

Reply:
Sorry, I missed the point in the sentence. I meant, Why do you return Selector<string> instead of Uri return type, if your property is returning ImageUrl?

Reply:
Oh, okay. The reason why the return type is 'Selector<>' is that user can set various image paths according to control states, such as Normal, Focused, and Pressed.
For example, in an application :
                IndicatorImageURL = new Selector<string>
                {
                    Normal = Tizen.Applications.Application.Current.DirectoryInfo.Resource + "normalImage.png",
                    Selected = Tizen.Applications.Application.Current.DirectoryInfo.Resource + "focusedImage.png"
                }

Reply:
@m-macias , could you check my comment please?
I'd like to get an accept the ACR and merge this patch by this week.